src/components/ResultViewer.js

```diff
@@ -36,6 +42,19 @@ export class ResultViewer extends React.Component {
     require('codemirror/keymap/sublime');
     require('codemirror-graphql/results/mode');
 
+    if (this.props.ResultsTooltip) {
+      require('codemirror-graphql/utils/info-addon');
+      CodeMirror.registerHelper(
+        'info',
+        'graphql-results',
```

Comment:
Second parametr to registerHelper is a bit confusing, but what it does is specify in which mode that helper should be used. So anything other than `graphql-results` would not work.